tests/test_experiment.py

```diff
@@ -0,0 +1,354 @@
+import pyspark.sql.functions as F
+import pytest
+
+from mozanalysis.experiment import Experiment
+from mozanalysis.utils import add_days
+
+
+def test_print_enrollment_window():
+    exp = Experiment('a-stub', '20190101')
+
+    # When we have complete data for 20190114...
+    the_fourteenth = '20190114'
+
+    # ...we have 14 dates of data for those who enrolled on the 1st
+    exp._print_enrollment_window(the_fourteenth, 14)
```

Comment:
this step seems to be a manual check. how about the helper return the list of dates and you can assert that it is the length you expect

Reply:
Ergh, this is a vestigial test. tdsmith correctly pointed out that the previous version of `test_print_enrollment_window` was mostly redundant due to `test_get_last_enrollment_date`, so I removed most of `test_print_enrollment_window`. I left this one statement to make sure that the method doesn't raise an exception (i.e. that the `format` parameters weren't mangled?).

But yeah it's barely worth testing, and in fact the `_print_enrollment_window()` method itself has shrunk over various refactorings to the point that I don't think it should exist any more. So I'm going to roll the print statement into `_get_last_enrollment_date()` and delete `_print_enrollment_window()` and `test_print_enrollment_window()`.